src/miner_lora.erl

```diff
@@ -375,6 +375,21 @@ select_gateway(Gateways) ->
             {ok, erlang:element(2, erlang:hd(maps:to_list(Gateways)))}
     end.
 
+%% Updates (IP, Port) for a given MAC, increments received packets from that MAC
+%% and creates gateway if necessary
+-spec update_gateway_record(map(), binary(), inet:ip_address(), inet:port_number()) -> gateway().
+update_gateway_record(
+    Gateways,
+    <<MAC:64/integer>>,
```

Comment:
Mac here should just be a standard param, no need for bit syntax here.  Bit syntax was already used to derive the value from the packet data in the caller.

Reply:
agree with this, as it could be semantics-changing.